stactools_planet/requirements.txt

```diff
@@ -1 +1,2 @@
-planet==1.4.6
\ No newline at end of file
+planet==1.4.6
+rasterio==1.1.8 --no-binary=rasterio
```

Comment:
Try putting rasterio first?

Reply:
I tried it, but i still get the same error message...

Reply:
Can you try changing this:

https://github.com/volaya/stactools/blob/planet_item_fixes/stactools_planet/setup.py#L21

to this:

https://github.com/volaya/stactools/blob/planet_item_fixes/stactools_planet/setup.py#L21

Reply:
> `line.split(' ')[0]`

That looks like it just removes the `--no-binary`?